components/entitlements/EntitlementsCard.js

```diff
@@ -91,30 +91,50 @@ export function EntitlementsCard(props){
                                                                             <p className="nfd-text-[#4A5567] nfd-font-normal nfd-mt-2">{__(entitlement.description, "wp-module-solutions" )}</p>                            
                                                                         </div>                
                                                                         {
-                                                                            activePluginsArray.includes(entitlement.basename) ?
+
+                                                                            entitlement.type === "plugin" ? 
                                                                             (
-                                                                                <Button
-                                                                                as="a"
-                                                                                className="nfd-button nfd-button--secondary nfd-self-center nfd-ml-auto nfd-font-normal nfd-text-[#000000]" 
-                                                                                href={ renderCTAUrl( entitlement.cta.url ) }
-                                                                                variant="secondary"
-                                                                                >
-                                                                                    { __(`${entitlement.cta.text}`, "wp-module-solutions") }
-                                                                                </Button>
+                                                                                activePluginsArray.includes(entitlement.basename) ?
+                                                                                (
+                                                                                    <Button
+                                                                                    as="a"
+                                                                                    className="nfd-button nfd-button--secondary nfd-self-center nfd-ml-auto nfd-font-normal nfd-text-[#000000]" 
+                                                                                    href={ renderCTAUrl(entitlement.cta.url) }
+                                                                                    variant="secondary"
+                                                                                    >
+                                                                                        { __(`${entitlement.cta.text}`, "wp-module-solutions") }
```

Comment:
@aratidgr8 - This isn't going to translate. You can't use dynamic values in a call to the `__` function. See https://ottopress.com/2012/internationalization-youre-probably-doing-it-wrong/